### DISCUSSION TITLE
Invite to join Phoenix Discord Channel

### DISCUSSION THREAD

**Discussion by abose** [phcode-dev/phoenix#77] [Announcements]
Hello everyone,

  Say hello to Pheonix devs and get in touch with our community via discord link: https://discord.com/invite/rBpTBPttca

Thanks,
Phoenix team.

**Comment by Crojav** [comment]
All I can say Thank you - This - Phoenix - help me!  I  Am  left with the memory - why can' t this editor not doing that what i think it should do. And there it is Phoenix.
![Uploading dolphin_002.png…]()

**Comment by Avinashm839** [comment]
Hi All
i am unable to set live preview in Phoenix Code

**Comment by onegoa** [comment]
Hi!
I'm experiencing an issue with the Phoenix Code program. When using a non-Latin (Cyrillic) keyboard layout, certain hotkeys (such as "Ctrl+S") don't work as expected. I attempted to remap the hotkeys to a combination like "Ctrl+Ы," which works fine in the Cyrillic layout, but it stops working when I switch back to the English (Latin) layout.

Could you please advise on how to make hotkeys function correctly across different keyboard layouts? Perhaps there's a configuration file or another solution to fix this issue.

Windows 11

**Comment by cebile447** [comment]
Hello, am lost I did all the coding procedure but my pictures on my website they don't show up.

**Reply by devvaannsh** [reply]
@cebile447, can you please share the code and also the project folder.
